app/lib/controlled_work_mappings/civ_means_7_new.yml

```diff
@@ -0,0 +1,457 @@
+    # Main home / client's share of total net equity (non-SMOD)
+    - name: FillText36
+      type: text
+      source: non_smod_main_home_net_equity
+
+    # Other property / client's share of total net equity (non-SMOD)
+    - name: FillText40
+      type: text
+      source: non_smod_additional_properties_net_equity
+
+    # Main home / final assessed amount of client's equity (non-SMOD)
+    - name: FillText41
+      type: text
+      source: non_smod_main_home_assessed_equity
+
+    # Other property / final assessed amount of client's equity (non-SMOD)
+    - name: FillText47
+      type: text
+      source: non_smod_additional_properties_assessed_equity
+
+    # Savings (non-SMOD)
+    - name: FillText110
+      type: text
+      source: non_smod_client_savings
+
+    # Valuables (non-SMOD)
+    - name: FillText112
+      type: text
+      source: non_smod_client_valuables
+
+    # Investments (non-SMOD)
+    - name: FillText111
+      type: text
+      source: non_smod_client_investments
+
+    # Total capital part B (non-SMOD)
+    - name: FillText118
+      type: text
+      source: combined_non_disputed_capital
+
+    # Total of all capital assets (Part A and Part B)
+    - name: FillText119
+      type: text
+      source: combined_assessed_capital
+
+- section: partner_capital
+  fields:
+
+    # Partner savings (non-SMOD)
+    - name: FillText114
+      type: text
+      source: partner_savings
+
+    # Partner investments (non-SMOD)
+    - name: FillText115
+      type: text
+      source: partner_investments
+
+    # Partner valuables (non-SMOD)
+    - name: FillText116
+      type: text
+      source: partner_valuables
+
+- section: income_even_if_passported
+  fields:
+    # The client is directly or indirectly in receipt of [passporting benefit]
+    - name: Passported
+      type: boolean_radio
+      source: passporting
+      yes_value: "Yes"
+      no_value: "No"
+
+- section: income
+  fields:
+    # Gross monthly earnings / Client
+    - name: FillText82
+      type: text
+      source: client_gross_income
+
+    # Benefits & Tax Credits / Client
+    - name: benefits_client
+      type: text
+      source: client_benefits
+
+    # Maintenance from former partner / Client
+    - name: maintenance_client
+      type: text
+      source: client_maintenance
+
+    # Pensions / Client
+    - name: pensions_client
+      type: text
+      source: client_pensions
+
+    # Student Finance / Client
+    - name: studentfinance_client
+      type: text
+      source: client_student_finance
+
+    # Family or Friends / Client
+    - name: friendsandfamily_client
+      type: text
+      source: client_friends_and_family
+
+    # Property or Lodger / Client
+    - name: property_client
+      type: text
+      source: client_property_lodger_income
+
+    # Other Income / Client
+    - name: otherincome_client
+      type: text
+      source: client_other_income
+
+    # Benefits & Tax Credits / Partner
+    - name: benefits_partner
```

Comment:
I'm wondering if these partner fields should go in the partner income section (line 402) but to be honest I'm not sure!

Reply:
I like grouping them this way as they are next to eachother on the CW form, but I understand why it might be better to move them. I will mull it over a bit.